pages/events/index.vue

```diff
@@ -161,6 +187,26 @@ export default class extends QiskitPage {
     return (this as any).filteredEvents.length === 0
   }
 
+  getOptions (type) {
```

Comment:
If you use a function to set the options, it will execute several times. Please, create a variable with the values as we did on line 176 and use it